README.md

```diff
@@ -1,55 +1,22 @@
 # schwifty-migration
 
-#### An [Objection.js](https://github.com/Vincit/objection.js) model migration tool for use during development
+#### An [Objection.js](https://vincit.github.io/objection.js) model migration tool for use during development
 
 [![Build Status](https://travis-ci.org/BigRoomStudios/schwifty-migration.svg?branch=master)](https://travis-ci.org/BigRoomStudios/schwifty-migration) [![Coverage Status](https://coveralls.io/repos/github/BigRoomStudios/schwifty-migration/badge.svg?branch=master)](https://coveralls.io/github/BigRoomStudios/schwifty-migration?branch=master) [![Security Status](https://nodesecurity.io/orgs/big-room-studios/projects/3f03e446-4689-49b3-9d82-ab3070eea553/badge)](https://nodesecurity.io/orgs/big-room-studios/projects/3f03e446-4689-49b3-9d82-ab3070eea553)
 
```

Comment:
Don't forget to add yourself as Lead Maintainer right about here.  Should be formatted,

> Lead Maintainer - [William Woodruff]()